src/operator/audit.ts

```diff
@@ -86,6 +86,9 @@ class AuditSubscriber<T, R> extends OuterSubscriber<T, R> {
         this.destination.error(errorObject.e);
       } else {
         this.add(this.throttled = subscribeToResult(this, duration));
```

Comment:
Refer this one - https://github.com/ReactiveX/rxjs/pull/1490/files#diff-4c9df56dc210f37759b96afb96e5fdd0R135

if `subscribeToResult` is synchronously subscribed, returned subscription is immediately unsubscribed already, so there's no meaning to add subscription for managing it (`this.add`). So in that case, check returned subscription and add only if it's asynchronous would be better.